tiny_dnn/core/kernels/maxpool_op_nnpack.h

```diff
@@ -12,9 +12,9 @@
 
 namespace tiny_dnn {
 namespace kernels {
-
-inline void maxpool_op_nnpack(const tensor_t &in_data,
-                              tensor_t &out_data,
+template <typename S1, typename S2>
```

Comment:
Please consider adding a new line like this
```cpp
namespace kernels {

template <typename S1, typename S2>
inline void.....
```
if doing so isn't against your policy.